packages/modules/core/src/Module.js

```diff
@@ -41,6 +41,7 @@ export const DigitUI = ({ stateCode, registry, enabledModules, moduleReducers })
     defaultOptions: {
       queries: {
         staleTime: Infinity,
+        cacheTime: 30 * 60 * 1000,
```

Comment:
needed as querycache is being used to pass function for revalidating inbox queries, otherwise after default time of 5 mins that function would be garbage collected from the cache,, with this it stays for 30 mins